client/me/sidebar/sidebar-item.jsx

```diff
@@ -1,38 +0,0 @@
-/**
- * External dependencies
- */
-var React = require( 'react' ),
-	classnames = require( 'classnames' );
-
-/**
- * Internal Dependencies
- */
-var layoutFocus = require( 'lib/layout-focus' ),
-	Gridicon = require( 'components/gridicon' );
-
-module.exports = React.createClass( {
-
-	displayName: 'MeSidebarItem',
-
-	onClick: function( event ) {
-		layoutFocus.setNext( 'content' );
-		window.scrollTo( 0, 0 );
-
-		if ( this.props.onClick ) {
-			this.props.onClick( event );
-		}
-	},
-
-	render: function() {
-		const selected = this.props.selected ? 'selected' : null;
-		return (
-			<li className={ classnames( this.props.className, selected, 'me-sidebar-item' ) } >
```

Comment:
There's a handful of `.me-sidebar-item` styles in `style.scss` that should probably be removed.